autoload/vital/__vimlparser__/VimlParser.vim

```diff
@@ -0,0 +1,5354 @@
+" vim:set ts=8 sts=2 sw=2 tw=0 et:
+"
+" VimL parser - Vim Script Parser
+"
+" License: This file is placed in the public domain.
+
+function! s:import() abort
+  return s:
+endfunction
+
+function! s:numtoname(num)
```

Comment:
`s:numtoname()` is used by `vimlparser#test()`

Reply:
Thanks. Done